www/src/js/components/installer.js

```diff
@@ -16,11 +16,12 @@ module.exports = React.createBackboneClass({
     var installActionString = model.get('installActionString');
     var isInstalled = model.get('isInstalled');
     var isInstallable = model.get('isInstallable');
+    var isPriced = model.get('priced');
     var installHTMLClass = model.get('installHTMLClass');
 
     var rootDivClass = "b-installer " + installerClass + " " + installHTMLClass;
 
-    if (isInstallable) {
+    if (isInstallable || isPriced) {
```

Comment:
Is that necessary? I don't see where the distinction is further needed in this file. Plus semantically, it feels like isPriced would always go with an isInstallable snap, shouldn't it ?